project-ivi/ivi
# Inital Testing

\#46

# Description

Been sitting on this for too long. Here's a few unit tests for evaluate. More tests on the way - they'll be different PRs

# Change Log
Describe all changes made 

-`.travis.yml` : include `jest` in process
- `README.md` : section on running tests
- `package.json` : jest config
- `src/../interpret.js` : commented out test code
- `test/evaluate_test.js` : 5 good-call unit tests for `evaulate()` function
- `test/helpers/input/variables.js` : variable declarations as code snippets for testing
- `test/helpers/output/variables.js` : JSON output of expected test results

# Manual Test Steps
Run `jest` to run all tests in `test` directory. `jest` will print results and generate code-coverage report. 
For continuous testing, use `npm run test` -  this will run all tests on any change to test code

Also see https://github.com/project-ivi/ivi/wiki/Testing for more info.
